modules/container_structure_test/1.15.0/MODULE.bazel

```diff
@@ -0,0 +1,17 @@
+"Bazel module definition, see https://bazel.build/external/overview#bzlmod"
+module(
+    name = "container_structure_test",
+    compatibility_level = 1,
+    # Replaced dynamically when published
+    version = "1.15.0",
+)
+
+# To run yq
+bazel_dep(name = "aspect_bazel_lib", version = "1.28.0")
```

Comment:
Bump this to 1.31.0? This has the latest version of yq.

Reply:
this is the minimum version we need (it has the skip-stardoc-under-bzlmod feature)